### ISSUE TITLE
Screen reader does not read project template group name in new project wizard

### ISSUE THREAD

**Issue by eharris369** [microclimate-dev2ops/microclimate-eclipse-tools#116]
The screen reader does not read the project template group name in the new project wizard.  Tabbing from the project name field goes directly to the filter text field and the reader just reads: "Type filter text".  It skips the "Project Template" group name so the user would not know what to do here.  Use a read only text field instead of a group name so that the reader picks it up.
